main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genre_objects_create = ["Western", "Action", "Dramma"]
+    for genre in genre_objects_create:
+        Genre.objects.create(name=genre)
+
+    actor_objects_create = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    for actor in actor_objects_create:
+        first_name_, last_name_ = actor[0], actor[1]
+        Actor.objects.create(
+            first_name=first_name_,
+            last_name=last_name_
+        )
+
+    Genre.objects.filter(
+        name="Dramma",
+    ).update(name="Drama")
+
+    actor_objects_update = [
+        (("George", "Klooney"), ("George", "Clooney")),
+        (("Kianu", "Reaves"), ("Keanu", "Reeves"))
+    ]
+    for actor in actor_objects_update:
```

Comment:
Use the same unpacking to make code more readable:
`for values_to_update, values in actor_objects_update:`